calc/src/desc.ts

```diff
@@ -524,7 +524,8 @@ function getEndOfTurn(
       damage -= gen > 6 ? Math.floor(defender.maxHP() / 32) : Math.floor(defender.maxHP() / 16);
       texts.push('reduced burn damage');
     } else if (!defender.hasAbility('Magic Guard')) {
-      damage -= gen > 6 ? Math.floor(defender.maxHP() / 16) : Math.floor(defender.maxHP() / 8);
+      damage -=
+        gen === 1 || gen > 6 ? Math.floor(defender.maxHP() / 16) : Math.floor(defender.maxHP() / 8);
```

Comment:
```suggestion
         Math.floor(defender.maxHP() / (gen === 1 || gen > 6 ? 16 : 8));
```

Reply:
I noticed some of the other "damage -=" calculations also could be simplified by moving Math.floor(defender.maxHP() / {value}) out of the conditional operators, so I went ahead and changed those too

Reply:
Thank you! Thats awesome!